packages/react/src/components/Dropdown/DropdownSearchInput.tsx

```diff
@@ -70,7 +73,7 @@ class DropdownSearchInput extends UIComponent<ReactProps<DropdownSearchInputProp
     }),
     accessibilityInputProps: PropTypes.object,
     accessibilityComboboxProps: PropTypes.object,
-    hasToggleButton: PropTypes.bool,
```

Comment:
In which cases were we using this before?

Reply:
we were not, forgot it there when we removed the `hasToggleButton` prop